frontend/src/elm/Pages/Prototype/Main.elm

```diff
@@ -0,0 +1,133 @@
+module Pages.Prototype.Main exposing (..)
+
+import Html exposing (Html, div, text, button)
+import Html.Events exposing (onClick)
+import Html.Attributes exposing (align, style)
+import Pages.Prototype.LoginBox as LoginBox
+import Pages.Prototype.FileSelectBox as FileSelectBox
+
+
+-- MODEL
+
+type alias Model =
+  { useLoginBox : Bool
+  , loginBox : LoginBox.Model
+  , fileSelectBox : FileSelectBox.Model
+  }
+
+
+init : ( Model, Cmd Msg )
+init =
+  ( { useLoginBox = True
+    , loginBox = LoginBox.init
+    , fileSelectBox = FileSelectBox.init
+    }
+  , Cmd.none )
+
+
+-- MESSAGES
+
+type Msg
+  = StartByLogin
+  | StartByFile
+  | LoginBoxMsg LoginBox.Msg
+  | FileSelectBoxMsg  FileSelectBox.Msg
+
+
+-- UPDATE
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+  case msg of
+    StartByLogin ->
+      ( { model | useLoginBox = True }, Cmd.none )
+
+    StartByFile ->
+      ( { model | useLoginBox = False }, Cmd.none )
+
+    LoginBoxMsg subMsg ->
+      let
+        ( newModel, cmd ) =
+          LoginBox.update subMsg model.loginBox
+      in
+        ( { model | loginBox = newModel }, Cmd.map LoginBoxMsg cmd )
+    
+    FileSelectBoxMsg subMsg ->
+      let
+        ( newModel, cmd ) =
+          FileSelectBox.update subMsg model.fileSelectBox
+      in
+        ( { model | fileSelectBox = newModel }, Cmd.map FileSelectBoxMsg cmd )
+
+
+-- SUBSCRIPTIONS
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
```

Comment:
Elm style guide suggests newline in every type declaration.
See "Types" section in : http://elm-lang.org/docs/style-guide